spec/workers/commit_monitor_handlers/commit_range/graphics_labeler_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe CommitMonitorHandlers::CommitRange::GraphicsLabeler do
+  let(:branch)         { create(:pr_branch) }
+  let(:git_service)    { double("GitService", :diff => double("RuggedDiff", :new_files => new_files)) }
+
+  before do
+    stub_sidekiq_logger
+    stub_settings(:graphics_labeler => {:enabled_repos => [branch.repo.name]})
+    expect_any_instance_of(Branch).to receive(:git_service).and_return(git_service)
```

Comment:
Another option is just to test process_branch directly. psuedo-code idea.  This is part of the reason most of the workers have a separate process_branch method from the perform method (although I never got around to writing the tests this way).

```ruby
subject = CommitMonitorHandlers::CommitRange::GraphicsLabeler.new
subject.branch = branch
allow(branch).to receive #...
```